server/openapi_server/util/configuration.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+defaultValues = {
+    "SQL_USER": "postgres",
+    "SQL_PASSWORD": "postgres",
+    "SQL_DB": "i2b2_data",
+    "SQL_HOST": "db",
+    "SQL_PORT": 5432,
+    "FLASK_PORT":"8081"
+}
+
+
+class Config(object):
```

Comment:
As mentioned previously, let's keep a single class. The name of this file and its class should be the same.